src/NWCClient.ts

```diff
@@ -181,6 +181,35 @@ export class NWCClient {
     return new NWCClient(options);
   }
 
+  static async getWalletServiceSupportedMethods(
```

Comment:
how about making on the instance? why do you think it should be on the class level?

```
client = NWCClient.withNewSecret(options);
await client.supportedMethods();
```

Reply:
That makes sense. Sorry, I originally thought we could request the methods just using the relay URL, but then they can't be filtered by the wallet service pubkey (just in case).

Thanks!